src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENTAL_COST = 40;
+  const MEDIUM_TERM_DISCOUNT = 3;
+  const LONG_TERM_DISCOUNT = 7;
+  const DISCOUNT_FOR_3_OR_MORE_DAYS = 20;
+  const DISCOUNT_FOR_7_OR_MORE_DAYS = 50;
+
+  const totalRentalCost = DAILY_RENTAL_COST * days;
+
+  if (days >= MEDIUM_TERM_DISCOUNT && days < LONG_TERM_DISCOUNT) {
+    return totalRentalCost - DISCOUNT_FOR_3_OR_MORE_DAYS;
+  }
+
+  if (days >= LONG_TERM_DISCOUNT) {
+    return totalRentalCost - DISCOUNT_FOR_7_OR_MORE_DAYS;
+  }
```

Comment:
The discount logic is correct, but it could be simplified. Since the discount for 7 or more days is greater than the discount for 3 or more days, you could combine the conditions into a single return statement. This would make the code cleaner and easier to maintain.